docs/user-guide/configuration/index.md

```diff
@@ -18,7 +18,11 @@ You can access information about properties by hovering over the property name.
             - <span class="value">validate-west</span></a>
 <a href="#shared"><span class="key">shared</span>:</a>
     <a href="#environment"><span class="key">environment</span>:
-        <span class="key">NODE_ENV</span>: <span class="value">test</span></a>
+    <span class="key">NODE_ENV</span>: <span class="value">test</span></a>
+    <a href="#settings"><span class="key">settings</span>:</a>
+        <a href="#email"><span class="key">email</span>:
+    <span class="key">addresses</span>: <span class="value">[test@email.com, test2@email.com]</span>
```

Comment:
When you click "View" on Github, it shows what @d2lam said
https://github.com/screwdriver-cd/guide/blob/670e8d97803e24102d52c60dace5ab22e2f4e3af/docs/user-guide/configuration/index.md